x/distribution/simulation/operations.go

```diff
@@ -110,16 +110,17 @@ func SimulateMsgWithdrawValidatorCommission(ak types.AccountKeeper, k keeper.Kee
 			return simulation.NoOpMsg(types.ModuleName), nil, nil
 		}
 
-		simAccount, found := simulation.FindAccount(accs, sdk.AccAddress(validator.GetOperator()))
-		if !found {
-			return simulation.NoOpMsg(types.ModuleName), nil, fmt.Errorf("validator %s not found", validator.GetOperator())
-		}
-
+		// FIXME: this msg is not simulated because the commission is always 0!
```

Comment:
cc: @alexanderbez idk what's going on here but the validator accumulated commission is always 0

Reply:
updated